src/interfaces/wallet.cpp

```diff
@@ -170,7 +193,7 @@ class WalletImpl : public Wallet
             *name = it->second.name;
         }
         if (is_mine) {
-            *is_mine = IsMine(*m_wallet, dest);
+            *is_mine = m_wallet->IsMine(dest);
```

Comment:
Please try to understand the code you're commenting on.

Reply:
@sipa: Clarifying what you're referring to would be both constructive and polite :)

Reply:
This line is merely changing how `IsMine` is called. In order to avoid difficulties for reviewers, the existing code (the assignment itself) is not being changed.